### ISSUE TITLE
Screen resolution missing 4K

### ISSUE THREAD

**Issue by evertvorster** [vegastrike/Vega-Strike-Engine-Source#145]
Just installed VegaStrike. 
vegasettings did not have the resolution available for my monitor, 3840x2160
So, I have been playing on HD resolution. 
Would this be terribly hard to fix?

**Comment by BenjamenMeyer** [comment]
@Loki1950 can you take a look?

**Comment by nabaco** [comment]
To add an option for that screen in terms of `vegasettings` is very simple. The question is, does the the engine support such a resolution.
@evertvorster you could try to experiment yourself, by adding the resolution in `~/.vegastrike/vegastrike.conf`, and report the results here.

**Comment by evertvorster** [comment]
Hi nabaco.
I had a quick play with it. Success! 

I had to add the resolution to the top line with all the resolutions, and down in the xml definitions as well. Attached is my modified file.

[vegastrike.config.zip](https://github.com/vegastrike/Vega-Strike-Engine-Source/files/4785232/vegastrike.config.zip)

**Comment by nabaco** [comment]
Thanks for @evertvorster for checking and testing this.
I'll open a PR shortly to add this by default.

**Comment by Loki1950** [comment]
Remember when LCD monitors first came out with wide aspects ratios www2 had access so he added the relevant resolutions they worked right away :) and was quickly committed to svn by klauss IIRC

**Comment by stephengtuggy** [comment]
Fixed in https://github.com/vegastrike/Assets-Production/pull/13 , I believe.

**Comment by BenjamenMeyer** [comment]
@evertvorster please confirm so we can close this.

**Comment by evertvorster** [comment]
Yes, for now this issue is sort of fixed. The proper way would be for there to be a couple of usual resolutions, but then the user should also be able to set their own resolutions, and/or read the available resolutions from the selected monitor.
